### ISSUE TITLE
separate each template format in submodules

### ISSUE THREAD

**Issue by marado** [ubuconeurope/2019sintra-presentation-template#3]
Having to clone this entire project is a bit of a waste, if you only care about the template in one of the formats. We should separate each format into its own git repository, and managing them here, by considering each of them git submodules.

**Comment by gsilvapt** [comment]
I disagree, although I'm open to help moving things if everyone else (@ubuconeurope/maintainers) agrees.

Our conclusions with this experiment is that not too many people are adhering that much to this. That would make sense if we had many people using them. I'm strongly against the spread of information because it does not help when we want people to adopt small things, like templates. You can still download the file from the repository, instead of cloning it. I understand this is a bit wasteful if you're talking about contributions but I feel this is a necessary evil at this stage.

Hopefully, future Ubucon events will have more adopters of this methodologies and then it will make more sense to split things into a major repo and their respective sub modules.

**Comment by tcarreira** [comment]
I agree with @gsilvapt, I don't see a strong reason to split it into several repositories. 

Nevertheless, I agree with the pain stated by @marado. 
We could add a release with assets as tar/zip by template. If you just want to use the template, just download the tar/zip and it's done. 

But I don't have it as priority right now, and probably won't do it until the ubucon.
If you agree, you may alter the title of this issue to match my description ;)
